### PR TITLE
EAGLE-643 Implement oozieAuditLogAppProvider

### PR DESCRIPTION
**Pull request by chitin**
EAGLE-643 Implement oozieAuditLogAppProvider
- Add provider for oozie audit log

https://issues.apache.org/jira/browse/EAGLE-643

### CODE REVIEW

File: eagle-security/eagle-security-oozie-auditlog/src/main/java/org/apache/eagle/security/oozie/parse/OozieAuditLogAppProvider.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.apache.eagle.security.oozie.parse;
```
Comment (haoch) [inline eagle-security/eagle-security-oozie-auditlog/src/main/java/org/apache/eagle/security/oozie/parse/OozieAuditLogAppProvider.java]: Missing license header

Comment (haoch) [inline eagle-security/eagle-security-oozie-auditlog/src/main/java/org/apache/eagle/security/oozie/parse/OozieAuditLogAppProvider.java]: Missing license header

Comment (chitin) [inline eagle-security/eagle-security-oozie-auditlog/src/main/java/org/apache/eagle/security/oozie/parse/OozieAuditLogAppProvider.java]: I'm going to fix it.

File: eagle-security/eagle-security-oozie-auditlog/src/main/resources/META-INF/providers/org.apache.eagle.security.oozie.parse.OozieAuditLogAppProvider.xml [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        <!-- topology related configurations -->
+        <property>
+            <name>workers</name>
+            <displayName>workers</displayName>
+            <value>1</value>
+            <description>number of topology workers</description>
+        </property>
+        <property>
+            <name>topology.numOfJoinTasks</name>
+            <displayName>topology.numOfJoinTasks</displayName>
```
Comment (haoch) [inline eagle-security/eagle-security-oozie-auditlog/src/main/resources/META-INF/providers/org.apache.eagle.security.oozie.parse.OozieAuditLogAppProvider.xml]: Please change `displayName` of all properties in configuration descriptor to **human-readable name**, for example:  topology.numOfJoinTasks -> "Join Tasks Num"

**Comment by haoch** [comment]
BTW: Could you please provide more description or screenshot about the feature? 

For example:

- Installation Dialog Screenshot
- Populated Stream Schema
- Any UI features?

**Comment by haoch** [comment]
LGTM

**Pull request closed without merging**